src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -1067,7 +1068,8 @@ void CacheManager::ResetIgnitionCycles() {
 
 int CacheManager::TimeoutResponse() {
   CACHE_MANAGER_CHECK(0);
-  return pt_->policy_table.module_config.timeout_after_x_seconds;
+  return pt_->policy_table.module_config.timeout_after_x_seconds *
```

Comment:
@LitvinenkoIra 
I think you don't have to convert value to ms here, since this method wraps getting value of variable called 'timeout_after_x_seconds' so value in seconds should be returned. Conversion has to be done in place where method is called.